### PR TITLE
Added FullPath to File for Zips.

### PR DESCRIPTION
**Pull request by J7mbo**
Hi @mholt, saw that you're busy with Caddy, all good I'll use my fork for a while. 

Related to #130.

When you have time, can this be considered? There's a lot missing, the `File` is used everywhere and I only needed it when walking around a Zip, but perhaps we can discuss making this work for everything and get the PR in.

### CODE REVIEW

**Comment by J7mbo** [comment]
If this isn't going anywhere I will close.

File: archiver.go [hunk 1]
```diff
@@ -103,6 +103,8 @@ type Extractor interface {
 // File provides methods for accessing information about
 // or contents of a file within an archive.
 type File struct {
+	FullPath string
```
Comment (mholt) [inline archiver.go:106]: Would be useful to provide a comment here explaining what the field should contain.

**Review by mholt — CHANGES_REQUESTED** [review]
Thanks, sorry for the delay.

**Comment by mholt** [comment]
I think this is implemented in #302, as NameInArchive, if I understand correctly.

**Comment by J7mbo** [comment]
All good, I didn't put too much effort into it. Good luck in 2022! 👋

**Pull request closed without merging**